apps/example-todo-app/pages/api/todo/list.ts

```diff
@@ -0,0 +1,28 @@
+import { checkRouteSpec, withRouteSpec } from "lib/middlewares"
+import { z } from "zod"
+
+export const commonParams = z.object({
+  ids: z.array(z.string().uuid()),
+})
+
+export const route_spec = checkRouteSpec({
+  methods: ["GET"],
+  auth: "auth_token",
+  commonParams,
+  jsonResponse: z.object({
+    ok: z.boolean(),
+    todos: z
+      .object({
+        id: z.string().uuid(),
+      })
+      .array(),
+  }),
+})
+
+export default withRouteSpec(route_spec)(async (req, res) => {
+  const { ids } = req.commonParams
+
+  console.log({ ids })
```

Comment:
```suggestion
```

Reply:
ehh maybe leave it since there are no tests here

Reply:
added a test